pages/index.html

```diff
@@ -10,7 +10,7 @@
     <li>
       <a href="https://twitter.com/share" class="twitter-share-button"
          data-url="http://www.mixxx.org"
-         data-text="{% blocktrans %}Check out Mixxx - Free MP3 DJ mixing software for Windows, Mac OS X, and Linux{% endblocktrans %}"
+         data-text="{% blocktrans %}Check out Mixxx - Free Digital DJ MP3 mixing software for Windows, Mac OS X, and Linux{% endblocktrans %}"
```

Comment:
I tried making the title "Free & Open Source Digital DJ MP3 Mixing Software", but that feels way too wordy and complicated. I think it should be left as "Free Digital DJ MP3 Mixing Software".